datacite-commons/src/main/java/no/unit/nva/doi/DataCiteDoiClient.java

```diff
@@ -60,30 +57,30 @@ public DataCiteDoiClient(HttpClient httpClient, SecretsManagerClient secretsMana
     }
 
     @Override
-    public URI generateDraftDoi(Resource resource) {
-        return attempt(() -> sendDraftDoiRequest(resource))
+    public URI generateDraftDoi(URI requestingCustomer) {
+        return attempt(() -> sendDraftDoiRequest(requestingCustomer))
```

Comment:
Nice, can we maybe name variable in the way which will also mean something like "this is who owns the doi". At the same time something like `customerOwningDoi` is also not perfect.